managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/EntryCacheManager.java

```diff
@@ -59,7 +63,8 @@
 
     protected static final double MB = 1024 * 1024;
 
-    private static final double evictionTriggerThresholdPercent = 0.98;
+    @Setter
+    private volatile double evictionTriggerThresholdPercent = 0.98;
```

Comment:
This is already defined in config with init value. Can we remove it from this class ？

Reply:
Fixed:  removed the config: evictionTriggerThresholdPercent

Reply:
Then we can keep static final